frontend/src/store/modules/tab.ts

```diff
@@ -3,12 +3,13 @@ import { v1 as uuidv1 } from "uuid";
 import { TabInfo, AnyTabInfo, TabState } from "../../types";
 import * as types from "../mutation-types";
 import { makeActions } from "../actions";
+import { t } from "../../plugins/i18n";
 
 export const getDefaultTab = () => {
   return {
     id: uuidv1(),
-    name: "Untitled Sheet",
-    isSaved: false,
+    name: t("sql-editor.untitled-sheet"),
```

Comment:
I would put this as a separate const outside since the below code only just references it.

Reply:
this doesn't get resolved. I meat

export const defaultTabName = t("sql-editor.untitled-sheet")

Reply:
> this doesn't get resolved. I meat
> 
> export const defaultTabName = t("sql-editor.untitled-sheet")

I don't know what's the difference between the two cases?

Reply:
Maybe reuse it somehow?

Reply:
It's in the place using it

instead of 

if (currentTab.value.name === getDefaultTab().name && !sheetName) {
}

we can just write

if (currentTab.value.name === defaultTabName && !sheetName) {
}

the latter is more straightforward.

Reply:
getDefaultTab() is a method, and it requires the reader to read getDefaultTab to see whether it has any side-effect.

And we are essentially comparing whether the name has been changed from the defaultName, so just comparing the name instead of comparing a name returned from a method is more readable.

Reply:
> getDefaultTab() is a method, and it requires the reader to read getDefaultTab to see whether it has any side-effect.
> 
> And we are essentially comparing whether the name has been changed from the defaultName, so just comparing the name instead of comparing a name returned from a method is more readable.

Get the point